src/components/Header/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+
+type Props = {
+  activeTodos: boolean;
+  newTodoField: React.RefObject<HTMLInputElement>;
+};
+
+export const Header: React.FC<Props> = ({
+  activeTodos,
+  newTodoField,
+
+}) => {
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
```

Comment:
you can move newTodoField to this component )

```suggestion
export const Header: React.FC<Props> = ({
  activeTodos,
}) => {
  const newTodoField = useRef<HTMLInputElement>(null);

  useEffect(() => {
    if (newTodoField.current) {
      newTodoField.current.focus();
    }
  }, []);
```